visualmetrics.py

```diff
@@ -36,11 +36,12 @@
 import shutil
 import subprocess
 import tempfile
+from ssim import compute_ssim
```

Comment:
Can we do this at the time of the actual ssim calculation so the dependency becomes a soft dependency and only if ssim-based SI is requested?